### PR TITLE
20523 add new global built in variables

### LINKED ISSUE #20523: Add new global built-in variables
**What would you like to be added**:
${__dashbord.name} : the dashboard name
${__org.name}: the current organization name
${__from.text}: the from variable in string format
${__to.text}: the to variable in string format

**Why is this needed**:
I need to add a title panel on each dashboard to understand on printed page what is visible.
I would like this: 
Overview : Nile (2019-11-19 13:30:40 to 2019-11-20 13:30:42)
With a Text panel with this:
`${__dashbord.name} : ${__org.name} (${__from.text} to ${__to.text})`

### PR DESCRIPTION
**Pull request by dcastanier**
What this PR does / why we need it:
It's an implementation of 'Add new global built-in variables #20523'

Which issue(s) this PR fixes:
Fixes #20523

Special notes for your reviewer:
This PR follow and replace #20533 that's had an issue with merge of previous changes

### CODE REVIEW

File: docs/sources/reference/templating.md [hunk 1]
```diff
@@ -311,6 +311,10 @@ Grafana has global built-in variables that can be used in expressions in the que
 Grafana has two built in time range variables in `$__from` and `$__to`. They are currently always interpolated
 as epoch milliseconds. These variables are only available in Grafana v6.0 and above.
 
+> Only available in Grafana v6.6+
+
+Grafana 6.6 and above allow to format time in string `${__from.text}` and `${__to.text}`, with default text format (`YYYY-MM-DD HH:mm:ss`)
```
Comment (oddlittlebird) [inline docs/sources/reference/templating.md]:
```suggestion
In Grafana 6.6 and above, you can format time in a string `${__from.text}` and `${__to.text}`, with default text format (`YYYY-MM-DD HH:mm:ss`).
```

File: docs/sources/reference/templating.md [hunk 2]
```diff
@@ -347,6 +351,18 @@ This variable is only available in the Singlestat panel and can be used in the p
 
 Currently only supported for Prometheus data sources. This variable represents the range for the current dashboard. It is calculated by `to - from`. It has a millisecond and a second representation called `$__range_ms` and `$__range_s`.
 
+### The $__dashboard Variable
+> Only available in Grafana v6.6+
+
+This variable is the uid of the current dashboard.
```
Comment (oddlittlebird) [inline docs/sources/reference/templating.md]: uid - Is this user ID?

File: docs/sources/reference/templating.md [hunk 3]
```diff
@@ -347,6 +351,18 @@ This variable is only available in the Singlestat panel and can be used in the p
...
+`${__dashboard.name}` is the name of the current dashboard.
+
+### The $__org Variable
+> Only available in Grafana v6.6+
+
+This variable is the id of the current organization.
```
Comment (oddlittlebird) [inline docs/sources/reference/templating.md]:
```suggestion
This variable is the ID of the current organization.
```

**Review by oddlittlebird — CHANGES_REQUESTED** [review]

File: docs/sources/reference/templating.md (hunk 2 above)
Comment (dcastanier) [reply docs/sources/reference/templating.md]: UID is Unique ID, a string like "1QWfqrLiz" that identify a dashboard. You could specify it when you import a dashboard. 
[https://grafana.com/docs/grafana/latest/reference/dashboard/](url)

**Comment by dprokop** [comment]
@dcastanier talked this through with the team today and we decided not to include *time formatting* in this PR. The thing is we want to think about how to best approach time variables formatting syntax, to make it more flexible. 

Could you please revert those changes and leave only those related to `__dashboard` and `__org` variables? Sorry about that!

**Comment by dcastanier** [comment]
this PR is remplaced by #21790 to resolve GIT merge issue

**Pull request closed without merging**